client/my-sites/site-settings/style.scss

```diff
@@ -160,6 +160,26 @@
 	}
 }
 
+.site-settings__general-settings {
+	.form-toggle__label {
+		display: flex;
+		align-items: center;
+	}
+
+	.form-toggle__switch {
+		flex: 1 0 auto;
+	}
+
+	.form-toggle__switch + span {
+		flex: 0 1 100%;
+		margin-left: 12px;
+	}
+
+	p.form-setting-explanation.is-indented {
```

Comment:
Regardless of whether `FormSettingExplanation` should be indented, this CSS should be removed. If you do end up needing to remove the indent, remove the `isIndented` property from the component.